sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/jsonpatch/JsonPatchDocument.java

```diff
@@ -0,0 +1,127 @@
+     */
+    public JsonPatchDocument appendAdd(String path, String rawJsonValue) {
+        operations.add(new JsonPatchOperation(JsonPatchOperationKind.ADD, path, null, rawJsonValue));
+        return this;
+    }
+
+    /**
+     * Appends a "replace" operation to this JSON Patch document.
+     *
+     * @param path The path to replace.
+     * @param rawJsonValue The raw JSON value to use as the replacement.
+     * @return The updated JsonPatchDocument object.
+     */
+    public JsonPatchDocument appendReplace(String path, String rawJsonValue) {
+        operations.add(new JsonPatchOperation(JsonPatchOperationKind.REPLACE, path, null, rawJsonValue));
+        return this;
+    }
+
+    /**
+     * Appends a "copy" operation to this JSON Patch document.
+     *
+     * @param from The path to copy from.
+     * @param path The path to copy to.
+     * @return The updated JsonPatchDocument object.
+     */
+    public JsonPatchDocument appendCopy(String from, String path) {
+        operations.add(new JsonPatchOperation(JsonPatchOperationKind.COPY, path, from, null));
+        return this;
+    }
+
+    /**
+     * Appends a "move" operation to this JSON Patch document.
+     *
+     * @param from The path to move from.
+     * @param path The path to move to.
+     * @return The updated JsonPatchDocument object.
+     */
+    public JsonPatchDocument appendMove(String from, String path) {
+        operations.add(new JsonPatchOperation(JsonPatchOperationKind.MOVE, path, from, null));
+        return this;
+    }
+
+    /**
+     * Appends a "remove" operation to this JSON Patch document.
+     *
+     * @param path The path to remove.
+     * @return The updated JsonPatchDocument object.
+     */
+    public JsonPatchDocument appendRemove(String path) {
+        operations.add(new JsonPatchOperation(JsonPatchOperationKind.REMOVE, path, null, null));
+        return this;
+    }
+
+    /**
+     * Appends a "test" operation to this JSON Patch document.
+     *
+     * @param path The path to test.
+     * @param rawJsonValue The raw JSON value to test against.
+     * @return The updated JsonPatchDocument object.
+     */
+    public JsonPatchDocument appendTest(String path, String rawJsonValue) {
+        operations.add(new JsonPatchOperation(JsonPatchOperationKind.TEST, path, null, rawJsonValue));
+        return this;
+    }
+
+    /**
+     * Gets a formatted JSON string representation of this JSON Patch document.
+     *
+     * @return The formatted JSON String representing this JSON Patch docuemnt.
+     */
+    @Override
+    public String toString() {
+        StringBuilder document = new StringBuilder().append("[");
+
+        for (JsonPatchOperation operation : operations) {
+            writeOperation(document, operation);
+        }
+
+        return document.append("]").toString();
+    }
+
+    private static void writeOperation(StringBuilder document, JsonPatchOperation operation) {
+        document.append("{")
```

Comment:
Need to look into using Jackson to generate this String as we may need to use JSON escaping.